app/views/estimate_flow/applicant.html.slim

```diff
@@ -23,12 +23,26 @@
                                           hint: { text: t(".over_60.hint") }
 
     - if FeatureFlags.enabled?(:partner)
-      = form.govuk_collection_radio_buttons :partner,
-                                              yes_no_options,
-                                              :first,
-                                              :last,
-                                              legend: { text: t(".partner.legend") },
-                                              hint: { text: t(".partner.hint") }
+      = form.govuk_radio_buttons_fieldset(:partner, legend: { text: t(".partner.legend") }, hint: lambda { \
+                                                      tag.p(t(".partner.hint_text"), class: "govuk-hint") + \
+                                                      tag.ul(class: "govuk-list govuk-list--bullet") do \
+                                                        t(".partner.hint_list").map { |benefit| \
+                                                          tag.li(benefit, class: "govuk-hint") \
+                                                        }.join.html_safe \
+                                                      end + \
+                                                      tag.p(t(".partner.partner_additional.hint_text"), class: "govuk-hint") + \
+                                                      tag.ul(class: "govuk-list govuk-list--bullet") do \
+                                                        t(".partner.partner_additional.hint_list").map { |benefit| \
+                                                          tag.li(benefit, class: "govuk-hint") \
+                                                        }.join.html_safe \
+                                                      end \
+                                                     }) do
+        - yes_no_options.each do |btn|
+          - rb_hint = btn[1] == "Yes" ? t(".partner.rb_hint_text") : ""
+          = form.govuk_radio_button :partner, btn[0],
+                  link_errors: true,
+                  label: { text: btn[1] },
+                  hint:  { text: rb_hint }
```

Comment:
Given that there are only 2 options in this list and multiple if conditions, IMHO this would be better as 2 separate statements.  (Actually there are 2 subtle defects as well - the Welsh translation fails (due to the 'Yes'), and 'link_errors' should only be set true for the first element)

Reply:
thank you